htdocs/install/mysql/tables/llx_webhook_target_extrafields.sql

```diff
@@ -0,0 +1,23 @@
+-- Copyright (C) ---Put here your own copyright and developer email---
```

Comment:
No need of extrafields feature. llx_webhook_target is just a "setup" table.
Remove this table.